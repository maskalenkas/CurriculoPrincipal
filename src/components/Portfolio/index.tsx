import * as S from './styles';

const Portfolio = () => (
  <S.Wrapper>
    <h1>Portfolio</h1>
  </S.Wrapper>
);

export default Portfolio;
