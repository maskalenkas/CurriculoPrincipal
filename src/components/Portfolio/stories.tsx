import { Story, Meta } from '@storybook/react/types-6-0';
import Portfolio from '.';

export default {
  title: 'Portfolio',
  component: Portfolio,
} as Meta;

export const Default: Story = () => <Portfolio />;
