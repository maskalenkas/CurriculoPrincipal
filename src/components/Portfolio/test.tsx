import { render, screen } from '@testing-library/react';

import Portfolio from '.';

describe('<Portfolio />', () => {
  it('should render the heading', () => {
    const { container } = render(<Portfolio />)

    expect(screen.getByRole('heading', { name: /Portfolio/i })).toBeInTheDocument()

    expect(container.firstChild).toMatchSnapshot()
  });
});
