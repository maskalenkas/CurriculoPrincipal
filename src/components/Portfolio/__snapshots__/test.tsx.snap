// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Portfolio /> should render the heading 1`] = `
<main
  class=""
>
  <h1>
    Portfolio
  </h1>
</main>
`;
